test/jest/unit/snyk-code/snyk-code-test.spec.ts

```diff
@@ -258,6 +258,46 @@ describe('Test snyk code', () => {
     ).rejects.toHaveProperty('userMessage', 'Test limit reached!');
   });
 
+  it('should create sarif result when `--sarif-file-output` is used', async () => {
+    const options: ArgsOptions = {
+      path: '',
+      traverseNodeModules: false,
+      showVulnPaths: 'none',
+      code: true,
+      _: [],
+      _doubleDashArgs: [],
+      'sarif-file-output': 'test',
```

Comment:
[nitpicking]  since sarif is a JSON-based format, i would name the file to reflect that it is indeed a json file
```suggestion
      'sarif-file-output': 'test.json',
```